src/pipecat/services/hathora/stt.py

```diff
@@ -0,0 +1,107 @@
+#
+# Copyright (c) 2024–2025, Daily
+#
+# SPDX-License-Identifier: BSD 2-Clause License
+#
+
+"""[Hathora-hosted](https://models.hathora.dev) speech-to-text services."""
+
+import os
+from typing import Optional
+
+import aiohttp
+from loguru import logger
+
+from pipecat.frames.frames import (
+    ErrorFrame,
+    TranscriptionFrame,
+)
+from pipecat.services.stt_service import SegmentedSTTService
+from pipecat.transcriptions.language import Language
+from pipecat.utils.time import time_now_iso8601
+
+class ParakeetTDTSTTService(SegmentedSTTService):
+    """Parakeet TDT is a multilingual automatic speech recognition model
```

Comment:
Nit: Docstring linting doesn't allow to wrap the first line in a docstring🤷 

```suggestion
    """Parakeet TDT is a multilingual automatic speech recognition model
```